src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotCallDangerousMethodsInDeserializationTests.cs

```diff
@@ -0,0 +1,1817 @@
+﻿using Microsoft.CodeAnalysis.Diagnostics;
```

Comment:
`// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.`